DependencyInjection/Configuration.php

```diff
@@ -57,12 +57,18 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->scalarNode('unauthorized_challenge')->defaultNull()->end()
                 ->arrayNode('param_fetcher_listener')
-                    ->validate()
-                        ->ifNotInArray(Configuration::$forceOptionValues)
-                        ->thenInvalid('The param_fetcher_listener option does not support %s. Please choose one of '.json_encode(Configuration::$forceOptionValues))
+                    ->beforeNormalization()
+                        ->ifString()
+                        ->then(function ($v) { return array('enabled' => $v); })
```

Comment:
I am not sure if that really is the best solution. Imho `enabled` should only be `true` or `false`. What do you think about adding a boolean child node `force` which could be set accordingly?